.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+# 0010 - Interest Bearing Channel Application
+
+# Status
+
+Accepted
+
+# Context
+
+For a permissionless state channel network (SCN) to be sustainable, incentive structures must exist for each class of system actor (service providers, service consumers, & liquidity providers).
+
+Our work generally presumes the existence of incentives that motivate service providers & consumers. The design work for us is in mechanisms to incentivize intermediaries (liquidity providers) into the network.
+
+# Prior Art
+
+The Bitcoin Lightning Network (LN) is an in-production payment channel network (PCN) with a fee mechanism.
+
+## Similarities to Nitro-SCN
+
+In a PCN, payments between A and B are routed through intermediaries I*1, ..., I_n, where balances are pairwise adjusted along the route. Capital in intermediary channels is locked for the \_duration of the payment* - ie, until either the payment completes or times out.
+
+In an SCN, collateral to secure channels between A and B is locked along a similar route of intermediaries, and closing balance adjustments between A and B are pairwise adjusted along the route. Capital in intermediary channels is locked for the _duration of the virtual channel_.
+
+## Differences from Nitro-SCN
+
+### Ecosystem / Context
+
+The Bitcoin ecosystem has a strong intrinsic incentive toward participating in LN, as LN is the most (only?) viable solution to scaling BTC denominated payments with security anchored to bitcoin.
+
+Further: holders of BTC have relatively few options for activating their capital to produce a yield. Assets in EVM environments can generate yield via any number of defi protocols, including Ethereum proof-of-stake (staking pools offer approx %4 yield with near zero overhead).
+
+### Technical
+
+In the dominant use case, PCN payments are brief. The capital lockup time for an individual payment is `O(n)`, where `n` is length of the chain of intermediaries. LN strives toward a mesh network topology so that
+
+- scale requirements can be met (a single hub would struggle to be involved in _all_ payments)
+- privacy claims are defensible (payments are onion routed - each node passes the payment one step further but cannot read the entire route. Would be moot with single hub, and less effective with dense network of hubs)
+
+By contrast, the dominant case in an SCN is for individual capital lockup events (channel fundings) to have non-trivial duration.
+
+For an SCN, the capital lockup time is equal to `fundingTime + channelRunningTime + defundingTime`. Each of `fundingTime` and `defundingTime` is expected to be smaller than LN's `O(n)`, because Nitro makes weaker privacy claims & is happy to disclose the entire intermediary chain at funding time. Despite this, the _total lockup time_ in Nitro will be much bigger on average because of the run-time of funded channels.
```

Comment:
I like this analysis. Out of scope for this ADR, but I'm wondering about (as a thought experiment) single-payment "disposable" virtual channels. They would have a very short `channelRunningTime`, but lean on the `O(1)`[^1] `(de)fundingTime` to offer advantages over LN / HTLC-PCNs. They are far quicker and less likely to fail. 

[^1]: Is this correct? If we eschew privacy as you suggest, and lean on say a centralized tracker / indexer / connection broker, do we really get this strong scaling improvement?

Reply:
These single-payment disposable virtual channels are more or less implemented as the [HashLockedSwap](https://github.com/statechannels/go-nitro/blob/main/nitro-protocol/contracts/examples/HashLockedSwap.sol) app.

I'm not sure I understand the conditional in your footnote. Our weaker privacy claims aren't a side effect of an indexer/pathbroker, but hard coded in terms of the `Participants` list of the channel's fixedPart, which all funding intermediaries can read in cleartext.

I think we should definitely have an eye toward implementing our best possible *simple payment*.